code/modules/vehicles/armored/__armored.dm

```diff
@@ -384,6 +407,20 @@
 		mod.on_equip(src, user)
 		return
 	if(interior) // if interior handle by gun breech
+		// check for easy loading instead
+		if(I.type in easy_load_list)
```

Comment:
easy return as well ? if type is not in list

Reply:
done